Build failure: ueberzugpp

### Steps To Reproduce

Steps to reproduce the behavior:
1. `nix build nixpkgs#ueberzugpp`

### Build log

```
error: builder for '/nix/store/cjfdi947ivzx2pv29d0xpql6rql63a5c-ueberzugpp-2.9.6.drv' failed with exit code 1;
       last 10 log lines:
       > -- Found WaylandProtocols: //nix/store/6w627naapy17padgxzkn6w0yzdyjgjwr-wayland-protocols-1.36/share/wayland-protocols (found version "1.36")
       > CMake Error at /nix/store/sa1yjb7xxrzcbd1bysj3r561vidfxxvb-cmake-3.29.6/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
       >   Could NOT find WaylandScanner (missing: WaylandScanner_EXECUTABLE)
       > Call Stack (most recent call first):
       >   /nix/store/sa1yjb7xxrzcbd1bysj3r561vidfxxvb-cmake-3.29.6/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
       >   /nix/store/kdbjfq8zvpzjjxpnp4r8xc65bs2bfn2f-extra-cmake-modules-5.116.0/share/ECM/find-modules/FindWaylandScanner.cmake:74 (find_package_handle_standard_args)
       >   CMakeLists.txt:150 (find_package)
       >
       >
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/cjfdi947ivzx2pv29d0xpql6rql63a5c-ueberzugpp-2.9.6.drv'.
```

full: https://gist.github.com/Familex/417e9c1b508620928049b6f7a92828cc

### Additional context

Occurred on last system update.

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.3, NixOS, 24.11 (Vicuna), 24.11.20240814.c3aa7b8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"home-manager-24.05.tar.gz, nixos-24.05"`
 - nixpkgs: `/nix/store/sfycwi72zfjsspidinx56ajaiffpyh17-source`
```
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ue/ueberzugpp/package.nix
+++ b/pkgs/by-name/ue/ueberzugpp/package.nix
@@ -24,6 +24,7 @@
 , extra-cmake-modules
 , wayland
 , wayland-protocols
+, wayland-scanner
 , enableX11 ? stdenv.isLinux
 , xorg
 }:
@@ -44,6 +45,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     cmake
     pkg-config
+  ] ++ lib.optionals enableWayland [
+    wayland-scanner
   ];
 
   buildInputs = [

